dj_hetmech_app/management/commands/populate_database.py

```diff
@@ -0,0 +1,91 @@
+"""
+```
+python manage.py makemigrations
+python manage.py migrate
+python manage.py flush --no-input
+python manage.py populate_database
+```
+"""
+
+import functools
+
+from django.core.management.base import BaseCommand
+import hetio.readwrite
+import pandas
+
+import dj_hetmech_app.models as hetmech_models
+
+
+class Command(BaseCommand):
+
+    help = 'Populate the database with Hetionet information'
+
+    @property
+    @functools.lru_cache()
+    def _hetionet_graph(self):
+        repo = 'https://github.com/hetio/hetionet'
+        commit = '23f6117c24b9a3130d8050ee4354b0ccd6cd5b9a'
+        path = 'hetnet/json/hetionet-v1.0.json.bz2'
+        url = f'{repo}/raw/{commit}/{path}'
+        return hetio.readwrite.read_graph(url)
+
+    @functools.lru_cache()
+    def _get_metanode(self, identifier):
+        """
+        Return the Django metanode object.
+        """
+        return hetmech_models.Metanode.objects.get(identifier=identifier)
```

Comment:
@dongbohu does this `_get_metanode` function belong in models? It seems that you can define functions for models